### PR TITLE
Fixes for issues FreeRDP#4571 and FreeRDP#5620

### PR DESCRIPTION
**Pull request by dragonuaix**
Just took solutions from these issues to pull.

### CODE REVIEW

File: client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java [hunk 1]
```diff
@@ -247,7 +247,7 @@ private void downgradeTables(SQLiteDatabase db)
 	private void insertDefault(SQLiteDatabase db)
 	{
 		ContentValues screenValues = new ContentValues();
-		screenValues.put(DB_KEY_SCREEN_COLORS, 32);
+		screenValues.put(DB_KEY_SCREEN_COLORS, 16);
```
Comment (akallabeth) [inline client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java]: why this change, nothing related to the one you announced.

File: client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java [hunk 2]
```diff
@@ -256,8 +256,8 @@ private void insertDefault(SQLiteDatabase db)
 		final long idScreen3g = db.insert(DB_TABLE_SCREEN, null, screenValues);
 
 		ContentValues performanceValues = new ContentValues();
-		performanceValues.put(DB_KEY_PERFORMANCE_RFX, 1);
-		performanceValues.put(DB_KEY_PERFORMANCE_GFX, 1);
+		performanceValues.put(DB_KEY_PERFORMANCE_RFX, 0);
```
Comment (akallabeth) [inline client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java]: why this change, nothing related to the one you announced.

File: client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java [hunk 3]
```diff
@@ -256,8 +256,8 @@ private void insertDefault(SQLiteDatabase db)
...
+		performanceValues.put(DB_KEY_PERFORMANCE_GFX, 0);
```
Comment (akallabeth) [inline client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java]: why this change, nothing related to the one you announced.

File: client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/presentation/SessionView.java [hunk 4]
```diff
@@ -334,6 +334,8 @@ public boolean onSingleTapUp(MotionEvent e)
 			sessionViewListener.onSessionViewBeginTouch();
 			switch (e.getButtonState())
 			{
+      		         // If it's 0 then it's normal touch, or BUTTON_PRIMARY then Samsung DeX or any other physical mouse left click.
+				case 0:
```
Comment (akallabeth) [inline client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/presentation/SessionView.java]: So, you tell android docs are wrong?
see `https://developer.android.com/reference/android/view/MotionEvent.html#getButtonState()`

I might be mistaken, but if you want support for physical devices you should listen for these events separately.

File: client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java [hunk 5]
```diff
@@ -293,6 +293,8 @@ private void insertDefault(SQLiteDatabase db)
 		bookmarkValues.put(DB_KEY_BOOKMARK_ASYNC_UPDATE, 1);
 		bookmarkValues.put(DB_KEY_BOOKMARK_CONSOLE_MODE, 0);
 		bookmarkValues.put(DB_KEY_BOOKMARK_DEBUG_LEVEL, "INFO");
+		bookmarkValues.put(DB_KEY_BOOKMARK_GW_HOSTNAME, "");
```
Comment (akallabeth) [inline client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java]: why this change, nothing related to the one you announced.

File: client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java [hunk 6]
```diff
@@ -293,6 +293,8 @@ private void insertDefault(SQLiteDatabase db)
...
+		bookmarkValues.put(DB_KEY_BOOKMARK_GW_PORT, "");
```
Comment (akallabeth) [inline client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java]: why this change, nothing related to the one you announced.

**Comment by dragonuaix** [comment]
Sorry, my first pull, not familiar wtih github logic yet.
Will close this one and create as it should be.

**Comment by akallabeth** [comment]
@dragonuaix well, thank you for the try then ;)
